src/components/BuildStatusCard.tsx

```diff
@@ -1,18 +1,6 @@
 import React from 'react';
 import moment from 'moment';
-import {
-  Avatar,
-  Badge,
-  Box,
-  Divider,
-  Flex,
-  Heading,
-  HStack,
-  Text,
-  VStack,
-  Icon,
-} from '@chakra-ui/react';
-import { BiTime } from 'react-icons/bi';
+import { Avatar, Badge, Box, Divider, Flex, Heading, HStack, Text, VStack } from '@chakra-ui/react';
 import { TimeIcon } from '@chakra-ui/icons';
 
 interface BuildStatusCardProps {
```

Comment:
maybe we could add a `type` field here, that we could define the different color mapping